runconfig/hostconfig.go

```diff
@@ -112,6 +112,7 @@ type HostConfig struct {
 	MemorySwap      int64  // Total memory usage (memory + swap); set `-1` to disable swap
 	CpuShares       int64  // CPU shares (relative weight vs. other containers)
 	CpusetCpus      string // CpusetCpus 0-2, 0,1
+	BlkioWeight     int64  // Block IO relative weight vs. other containers
 	Privileged      bool
```

Comment:
can you please put `relative weight vs. other containers` between `()` to be consistent ?